agent/bench-scripts/pbench-fio.md

```diff
@@ -200,21 +200,21 @@ Then you construct a fio job file for your initial sequential tests, and this wi
 
     [global]
     # size of each FIO file (in MB)
-    size=$@
+    size=1GiB
     # do not use gettimeofday, much too expensive for KVM guests
     clocksource=clock_gettime
     # give fio workers some time to get launched on remote hosts
     startdelay=5
     # files accessed by fio are in this directory
-    directory=$@
+    directory=/mnt/fio/files
     # write fio latency logs in /var/tmp with "fio" prefix
-    write_lat_log=/var/tmp/fio
+    write_lat_log=fio
     # write a record to latency log once per second
     log_avg_msec=1000
     # write fio histogram logs in /var/tmp/ with "fio" prefix
-    write_hist_log=/var/tmp/fio
+    #write_hist_log=/var/tmp/fio
     # write histogram record once every 10 seconds
-    log_hist_msec=10
+    #log_hist_msec=10
```

Comment:
Why do we need to comment out lines 215 and 217?

Reply:
These options were throwing errors. I did not investigate further.